src/onefetch/cli.rs

```diff
@@ -236,6 +252,11 @@ impl Cli {
         let no_color_palette = matches.is_present("no-color-palette");
         let print_languages = matches.is_present("languages");
         let print_package_managers = matches.is_present("package-managers");
+        let format = if let Some(format) = matches.value_of("format") {
+            String::from(format)
+        } else {
+            String::from("human")
+        };
```

Comment:
Since "format" has a default value, I think `matches.value_of("format")` will *always* be `Some`.
```suggestion
        let format = matches.value_of("format").map(String::from).unwrap();
```